Dockerfile.intel

```diff
@@ -31,29 +31,27 @@ ENV I_MPI_ROOT='/opt/intel/oneapi/mpi/latest' \
     CMAKE_PREFIX_PATH='/opt/intel/oneapi/vpl/latest:/opt/intel/oneapi/tbb/latest/env/..:/opt/intel/oneapi/dnnl/latest/cpu_dpcpp_gpu_dpcpp/../lib/cmake:/opt/intel/oneapi/dal/latest:/opt/intel/oneapi/compiler/latest/linux/IntelDPCPP' \
     CMPLR_ROOT='/opt/intel/oneapi/compiler/latest'
 
-# Using the Intel Compilers with mpi wrapper.
-# Further test on oneAPI Compilers needed: icx, icpx, ifx
-ENV CC=/opt/intel/oneapi/mpi/latest/bin/mpiicc \
-    CXX=/opt/intel/oneapi/mpi/latest/bin/mpiicpc \
-    FC=/opt/intel/oneapi/mpi/latest/bin/mpiifort
+ENV CC=mpiicc CXX=mpiicpc FC=mpiifort
```

Comment:
Still icpc/icc but not icpx/icx here?

Reply:
> Still icpc/icc but not icpx/icx here?

Here is the MPI wrapper, and icpx are enabled below. plz see

https://github.com/deepmodeling/abacus-develop/blob/bced11dcd8b9dc71d04198a24a9080ac60294a95/Dockerfile.intel#L54